### PR TITLE
Draw Regular Polygon

### PR DESCRIPTION
**Pull request by bettman-latin**
Hello! 
I'll start off by saying that I am new to git hub and this whole process (I have done this for a first commit project in school) so if there is anything I need to do differently please let me know, I am excited to learn, and I would be happy to make changes!

I have written a function that draws any regular polygon with n sides (given coordinate, width, and number of sides.
The function can also take style (filled, not filled...) and it can take degrees (if you want to rotate the polygon)

I have also added a test file and the test files' output pdf. I have tried to follow the style of other test files in this repository, but I may not have done it perfectly so let me know if there is anything you would like changed. 

Once again, if there are changes that need to be made/things I should be doing differently, let me know and I am excited to learn and make those changes.

### CODE REVIEW

File: fpdf/fpdf.py [hunk 1]
```diff
@@ -1054,6 +1054,41 @@ def circle(self, x, y, r, style=None):
...
+    @check_page
+    def regular_polygon(self, x, y, numSides, polyWidth, rotateDegrees=0, style=None):
+        """
+        Outputs a regular polygon with n sides
+        It can be rotated
+        Style can also be applied (fill, border...)
+
+        Args:
+            x (int): Abscissa of upper-left bounging box.
+            y (int): Ordinate of upper-left bounging box.
```
Comment (gmischler) [inline fpdf/fpdf.py:1066]: @bettman-latin, probably just an oversight: The correct spelling would be "bounding box"... :wink:

**Comment by Lucas-C** [comment]
Apart from the minor conflict in `fpdf.py` (that should easily be fixed by rebasing your branch on the upstream `master` branch) and @gmischler minor comment, this seems ready to be merged! :)

@bettman-latin: do you want to perform those 2 final steps, or would you like me to take over to finish things here?

**Comment by bettman-latin** [comment]
@Lucas-C Sorry for taking so long to follow up with this.  I read the documentation you sent and I have been trying to do the branch merge thing for a few days, but I can't seem to figure it out. I must have messed something up along the way, and I am not familiar enough with git/github to troubleshoot it myself without breaking it further. So, I can give it another shot if you want, but if you are willing, I think it might be best if you finish it up.

**Comment by Lucas-C** [comment]
@allcontributors please add @bettman-latin for code

**Comment by Lucas-C** [comment]
FYI I mentioned you in this blog article 😉
https://chezsoi.org/lucas/blog/hacktoberfest-on-fpdf2.html

**Pull request merged**